hq/app/service/CacheService.scala

```diff
@@ -0,0 +1,58 @@
+package service
+
+import akka.actor.ActorSystem
+import com.gu.Box
+import config.Config
+import model._
+import play.api.Configuration
+import rx.lang.scala.Observable
+import utils.attempt.{Attempt, FailedAttempt}
+
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration._
+
+class CacheService(config: Configuration) {
+  private val accounts = Config.getAwsAccounts(config)
+
+  def update[T](actorSystem: ActorSystem)(awsWorkerFn: (List[AwsAccount]) => Attempt[T])(boxUpdaterFn: Attempt[T] => Attempt[Unit])(implicit ec: ExecutionContext) = {
+    val cacheInterval = Config.getAwsApiCacheInterval(config)
```

Comment:
Is it worth bringing this out of the method to join `accounts` at the class level, for consistency? (line 15)
```
private val cacheInterval = Config.getAwsApiCacheInterval(config)
private val accounts = Config.getAwsAccounts(config)
```